packages/contracts-bedrock/test/L1/DelayedVetoable.t.sol

```diff
@@ -216,6 +216,31 @@ contract DelayedVetoable_HandleCall_TestFail is DelayedVetoable_Init {
         success2;
     }
 
+    function testFuzz_handleCall_forwardingTargetRetValue_succeeds(
+        bytes calldata inData,
+        bytes calldata outData
+    )
+        external
+    {
+        assumeNoClash(inData);
+
+        // Initiate the call
+        vm.prank(initiator);
+        (bool success,) = address(delayedVetoable).call(inData);
+        success;
+
+        vm.warp(block.timestamp + operatingDelay);
+        vm.expectEmit(true, false, false, true, address(delayedVetoable));
+        emit Forwarded(keccak256(inData), inData);
+
+        vm.mockCall(target, inData, outData);
+
+        // Forward the call
+        (bool success2, bytes memory retData) = address(delayedVetoable).call(inData);
+        assertTrue(success2);
+        assertEq(keccak256(retData), keccak256(outData));
```

Comment:
this test is quite similar to the test above, except it checks the data in the success rather than revert case.